SegmentEditorMaskVolume/SegmentEditorMaskVolumeLib/SegmentEditorEffect.py

```diff
@@ -44,19 +45,58 @@ def setupOptionsFrame(self):
     self.operationRadioButtons.append(self.fillOutsideButton)
     self.buttonToOperationNameMap[self.fillOutsideButton] = 'FILL_OUTSIDE'
 
+    self.binaryMaskFillButton = qt.QRadioButton("Binary mask")
+    self.operationRadioButtons.append(self.binaryMaskFillButton)
+    self.buttonToOperationNameMap[self.binaryMaskFillButton] = 'FILL_BOTH'
```

Comment:
Change this to something more explicit (if somebody just sees "FILL_BOTH" it is not clear what "both" refers to). It could be FILL_INSIDE_AND_OUTSIDE or FILL_COMPLETE, ....